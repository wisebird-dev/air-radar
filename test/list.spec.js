import React from 'react';
import {shallow} from 'enzyme';

import List from '../src/components/list';

test('it works', () => {
	const wrapper = shallow(<List/>);
	expect(wrapper).toMatchSnapshot();
});
