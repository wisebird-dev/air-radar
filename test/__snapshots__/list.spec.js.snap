// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
	position: relative;
	list-style: none;
	margin-left: 0;
	padding-left: 1.2em;
	padding-bottom: 12px;

	li:before {
		content: \\"â€º\\";
		position: absolute;
		left: 0;
	}
",
        ],
      },
      "displayName": "styled.ul",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "ul",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
